miriway

```diff
@@ -1,6 +1,22 @@
 #! /bin/sh
 set -e
 
+### Gtk Theming Tweaks
+config="${XDG_CONFIG_HOME:-$HOME/.config}/gtk-3.0/settings.ini"
+if [ ! -f "$config" ]; then exit 1; fi
+
+cursor_theme="$(grep 'gtk-cursor-theme-name' "$config" | sed 's/.*\s*=\s*//')"
+font_name="$(grep 'gtk-font-name' "$config" | sed 's/.*\s*=\s*//')"
+gtk_theme="$(grep 'gtk-theme-name' "$config" | sed 's/.*\s*=\s*//')"
+icon_theme="$(grep 'gtk-icon-theme-name' "$config" | sed 's/.*\s*=\s*//')"
```

Comment:
Currently i myself dont know maybe it will break miriway or it will bypass it and execute miriway, i cant be sure since i havent faced such situation yet !

Reply:
Improvements are Welcome...

Reply:
> What happens if these entries are not found?

Can you suggest me something to put (like condition) for else with the if statement, because adding a good argument to else will ensure the proper working of miriway, i mean if the file is found, execute gsettings operation else skip this operation and execute miriway as normal, i couldnt think of specifying such statement to else which could skip the gsettings part if the configs aren't found, maybe you can help ??

Reply:
Well i tested this and nothing bad actually happens,  in the most uncertain situation the script will simply ignore the bloc and move ahead with the following error :-

grep: /home/sam/.config/gtk-4.0/settings.ini: No such file or directory
grep: /home/sam/.config/gtk-4.0/settings.ini: No such file or directory
grep: /home/sam/.config/gtk-4.0/settings.ini: No such file or directory
grep: /home/sam/.config/gtk-4.0/settings.ini: No such file or directory
grep: /home/sam/.config/gtk-4.0/settings.ini: No such file or directory

Sam

Resolving this !!